jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -123,76 +89,52 @@ def test_status(self):
 
     def test_contents(self):
         contents = loads(self.resp.content.decode('utf-8'))
-        expected = dict(
-            applicant_id=13,
-            batch_number=9,
-            cnpj_cpf='11111111111111',
-            congressperson_document=2,
-            congressperson_id=1,
-            congressperson_name='Roger That',
-            document_id=42,
-            document_number='6',
-            document_type=7,
-            document_value=8.90,
-            installment=7,
-            issue_date='1970-01-01',
-            leg_of_the_trip='8',
-            month=1,
-            party='Partido',
-            passenger='John Doe',
-            all_reimbursement_numbers=[10, 11],
-            all_reimbursement_values=[12.13, 14.15],
-            all_net_values=[1.99, 2.99],
-            remark_value=1.23,
-            state='UF',
-            subquota_description='Subquota description',
-            subquota_group_description='Subquota group desc',
-            subquota_group_id=5,
-            subquota_id=4,
-            supplier='Acme',
-            term=1970,
-            term_id=3,
-            total_net_value=4.56,
-            total_reimbursement_value=None,
-            year=1970,
-            probability=0.5,
-            suspicions=suspicions,
-            receipt=dict(fetched=False, url=None)
-        )
-        self.assertEqual(expected, contents)
+        for result_attr, result_value in contents.items():
+            if not hasattr(self.reimbursement, result_attr):
+                continue
+            expected_value = getattr(self.reimbursement, result_attr)
+            self.assertEqual(str(result_value), str(expected_value))
```

Comment:
In addition to my former comment: 

The test you're suggesting is indeed interesting, but as a `test_context` (i.e. testing the data we're getting from the model and passing to the template). But with the class-based view from DRF this step is abstracted (and consequentially not tested).

What `test_contents` is actually testing is if the JSON output is what our frontend expects. So even if we change some attribute name, we can do some tweaks to avoid breaking compatibility with the frontend (i.e. tests will pass because JSON won't change) — or, surely better, have a failing test to remind us we've broken something and the JSON is not what we expected it to be anymore, so we can adapt the frontend and our expectations….